internal/lightcone/preservation/thisisme/thisisme.go

```diff
@@ -0,0 +1,72 @@
+package thisisme
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	mod    key.Modifier = "this-is-me"
+	modUlt key.Modifier = "this-is-me-ult"
+)
+
+type State struct {
+	ultBonus float64
+	idMap    map[key.TargetID]bool
+}
+
+// Increases the wearer's DEF by 16%/20%/24%/28%/32%. Increases the DMG of the
+// wearer when they use their Ultimate by 60%/75%/90%/105%/120% of the wearer's
+// DEF. This effect only applies 1 time per enemy target during each use of the
+// wearer's Ultimate.
+func init() {
+	lightcone.Register(key.ThisIsMe, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_PRESERVATION,
+		Promotions:    promotions,
+	})
+
+	modifier.Register(modUlt, modifier.Config{
```

Comment:
listeners should be on `mod` instead of `modUlt` (`modUlt` should just be deleted)